FracCalc/src/fracCalc/FracCalc.java

```diff
@@ -16,13 +30,334 @@ public static void main(String[] args)
+        		} else {
+        			numerator -= denominator;
+        		}
+        	}
+        	return denominator;
+        } else {
+        	return 1;
+        }
 
-    // TODO: Fill in the space below with any helper methods that you think you will need
+   }
+   
+   
+    /* separates the operand into whole number, numerator, and denominator and returns these values inside an array */
+    public static int[] separateOperand(String operand) {
+    	//initializes the variables
+    	String operandHold = operand;
+        String whole = "";
+        String numerator = "";
+        String denominator = "";
+        int spaceLoc = 0;
+        
+        //determines if there is a whole number- if there is not, it is set to 0
+        if (operandHold.contains("_")) { //if the operand has a whole number
+        	//separates the whole number
+        	spaceLoc = operandHold.indexOf("_");
+        	whole = operandHold.substring(0, spaceLoc);
+        	operandHold = operandHold.replaceFirst(whole, "");
+    		
+        	//separates the numerator
+    		spaceLoc = operandHold.indexOf("/");
+    		numerator = operandHold.substring(0, spaceLoc);
+    		operandHold = operandHold.replaceFirst(numerator, "");
+    		
+    		//separates the denominator
+    		operandHold = operandHold.replaceFirst("/", "");
+    		denominator = operandHold;
+    		
+        } else { //if operand does not have a whole number
+        	whole = "0";
+        	if (operandHold.contains("/")) { //if the value is a mixed number
+        		//separates the numerator
+        		spaceLoc = operandHold.indexOf("/");
+        		numerator = operandHold.substring(0, spaceLoc);
+        		operandHold = operandHold.replaceFirst(numerator, "");
+        		
+        		//separates the denominator
+        		operandHold = operandHold.replaceFirst("/", "");
+        		denominator = operandHold;
+
+        	} else { //if operand is 0 (0 divided by any number is still zero)
+        		numerator = "0";
+        		denominator = "1";
+        		whole = operand;
+        	}
+        	
+        }
+        
+        numerator = numerator.replaceFirst("/", "");
+        numerator = numerator.replaceFirst("_", "");
+        
+        //changes the variables into integers that can be worked with
+        int intWhole = operandIntoInt(whole);
+        int intNumerator = operandIntoInt(numerator);
+        int intDenominator = operandIntoInt(denominator);
+        int[] separatedOperand = {intWhole, intNumerator, intDenominator}; /* sets the separated operand 
+        into designated indexes of an array so as to return the whole operand with a single method */
+        return separatedOperand;
+    }
+    
+    
+    //changes the operand into an integer to do calculations with
+    public static int operandIntoInt(String operand) { 
+    	int intOperand = Integer.parseInt(operand);
+    	return intOperand;
+    }
+    
+    
+    /* multiplies the first and second operands */
```

Comment:
from a class-layout perspective, I would have kept all your math methods next to each other in the class.